jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/async/AsyncPool.java

```diff
@@ -51,7 +51,13 @@ public AsyncTask submit(Runnable task, String displayName, DataService dataServi
         synchronized(mutex) {
             String taskId = Long.toString(++counter) ;
             Fuseki.serverLog.info(format("Task : %s : %s",taskId, displayName)) ;
-            Callable<Object> c = Executors.callable(task) ;
+            Callable<Object> c = ()->{
+                try { task.run(); } 
+                catch (Throwable th) {
+                    Fuseki.serverLog.warn(format("Exception in task %s execution", taskId), th);
```

Comment:
Does this qualify as an error? (Logging-wise)

Reply:
Maybe. It is a development bug!